recipes/solweig-gpu/meta.yaml

```
@@ -0,0 +1,63 @@
+package:
+  name: solweig-gpu
+  version: "1.0.1"
+
+source:
+  url: https://files.pythonhosted.org/packages/de/83/0d408ff5f63b0f42441899e202b444c12a4326179ed352f4fc6534a11969/solweig_gpu-1.0.1.tar.gz
+  sha256: 437b78bae73ba49ac1c7dfeeb8f2176e7f674fabca4450c4c1fc6ccdd93f3b31
+
+build:
+  script: python -m pip install . --no-deps -vv
```

```suggestion
  entry_points:
    - thermal_comfort = solweig_gpu:thermal_comfort
  script: python -m pip install . --no-deps -vv
```

This may need some tweaking, but it should be something like this.